.scalafmt.conf

```diff
@@ -0,0 +1,18 @@
+version = 2.3.2
+style = default
+maxColumn = 120
```

Comment:
Wondering if we can set 140. I personally find 120 too... vertical.

Reply:
The recommendation is 80. I think 100-120 is a good compromise. (I personally find 140 a bit too horizontal.)